core/modules/layout/includes/block.hero.inc

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/**
+ *  BlockHero extends BlockText
+ *
+ *  This class allows us to create hero blocks with image backgrounds.
+ */
+class BlockHero extends BlockText {
+  /**
+   * {@inheritdoc}
+   */
+  function __construct($plugin_name, array $data) {
+    parent::__construct($plugin_name, $data);
+
+    $this->settings += array(
+      'image' => '',
+      'image_path' => '',
+    );
+  }
+
+  /**
+   *  Sets title text on draggable block panel in Layout builder.
+   */
+  function getAdminTitle() {
+    return !empty($this->settings['title']) ? check_plain($this->settings['title']) : t('Hero block');
+  }
+
+  /**
+   *  Builds the block's configuration form.
+   */
+  function form(&$form, &$form_state) {
+    parent::form($form, $form_state);
```

Comment:
As mentioned in the issue, if reusable isn't working yet we should disable it here with:
```
$form['convert']['#access'] = FALSE;
```